tools/packaging/kata-deploy/Dockerfile

```diff
@@ -2,30 +2,22 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-# Specify alternative base image, e.g. clefos for s390x
-ARG BASE_IMAGE_NAME=ubuntu
-ARG BASE_IMAGE_TAG=20.04
+ARG BASE_IMAGE_NAME=alpine
+ARG BASE_IMAGE_TAG=3.18
 FROM $BASE_IMAGE_NAME:$BASE_IMAGE_TAG
-ENV DEBIAN_FRONTEND=noninteractive
 ARG KATA_ARTIFACTS=./kata-static.tar.xz
 ARG DESTINATION=/opt/kata-artifacts
+ARG ARCH
```

Comment:
It seems this is needed to run the following command, right ?

```
curl -fL --progress-bar -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/${ARCH}/kubectl
```

Couldn't the ARCH be inferred from the alpine image with `uname -m` and converted to `amd64`  if needed ?
It would be safer to make the conversion explicit where it is needed  over getting it from the caller IMHO.